src/main/java/seedu/address/model/contact/TelegramHandle.java

```diff
@@ -5,12 +5,13 @@
 
 /**
  * Represents a Contact's telegram handle in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidTelegram(String)}
+ * Guarantees: immutable; is valid as declared in {@link #isValidTelegramHandle(String)}
  */
 public class TelegramHandle {
     public static final String MESSAGE_CONSTRAINTS =
-            "Telegram handles should not contain @, and it should have a length of 3 to 29 characters inclusive";
-    public static final String VALIDATION_REGEX = "^[A-Za-z0-9_]{3,29}$";
+            "Telegram handles should not contain @, and it should have a length of 5 to 32 characters "
+                    + "inclusive, containing alphabets, numbers and/or underscores only";
```

Comment:
I notice there is unclear grammar. A full stop will be clearer
"...32 characters inclusive. It must contain ... only"

Reply:
Have pushed a commit on this branch, not sure if it will show

Reply:
ok it has shown